packages/format-library/src/link/inline.js

```diff
@@ -230,7 +230,7 @@ function InlineLinkUI( {
 	return (
 		<Popover
 			anchor={ popoverAnchor }
-			focusOnMount={ focusOnMount.current }
+			focusOnMount={ focusOnMountWithTrap }
```

Comment:
If this is all we need, we can basically remove this line because it's the prop's default value.

On the other hand though, it kind of seems to me like the desired behavior here is actually `addingLink ? 'firstElement' : true`.